.github/ISSUE_TEMPLATE/support-request.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Support Request
+about: Report an issue faced while using Presto
+title: '[SUPPORT]'
+labels: support
+
+---
+
+**_Tips before filing an issue_**
+
+- Join the Presto Slack Channel to engage in conversations and get faster support at https://https://prestodb.io/slack.
+
+- If you have triaged this as a bug, then file an [BUG](https://github.com/prestodb/presto/issues/new/choose) directly.
+
+**Describe the problem you faced**
+
+* A clear and concise description of the problem.
+* Was this working before or is this a first try?
+* If this worked before, then what has changed that recently?  
+* Provide table DDLs and `` EXPLAIN ANALYZE `` for your Presto queries.
+
+
+**Steps To Reproduce**
+
+Steps to reproduce the behavior:
+1.
+2.
+3.
+4.
+
+**Expected behavior**
```

Comment:
This is a general question I had comparing the formatting of this file compared to the other files - something to think about and perhaps no change. 

In bug-report and feature-request, the header is formatted 
## Expected Behavior
(two hashtags, a space, then text)

but in the support-request file the similar header is formatted 
**Expected behavior**
(two asterisks, words, two asterisks)

Is there a reason that this template is formatted differently from the other files? Consider if reformatting this file in the same way as the others is appropriate. It might not be depending on the reason why it is different.

Reply:
Good catch, i have changed it and made it same across the board.